.changeset/seven-dolphins-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-jenkins-backend': minor
```

Comment:
Since this is not breaking (i think?) and the package is not yet 1.0, this can be `patch` level

Reply:
yes it is not breaking, changed to patch